src/main/java/org/broadinstitute/hellbender/utils/nio/SeekableByteChannelPrefetcher.java

```diff
@@ -441,6 +447,13 @@ public boolean isOpen() {
     @Override
     public void close() throws IOException {
         if (open) {
+            exec.shutdown();
+            try {
+                exec.awaitTermination(3, TimeUnit.MINUTES);
```

Comment:
Looking at the docs you're right - shutdown doesn't interrupt the thread so we need shutdownNow! But there's no way in Java to "murder" a thread, interrupting is the closest we can get. That means it's still worthwhile to wait for the worker to process its interrupted state (using awaitTermination).